### ISSUE TITLE
Mobile/Android not typing numbers

### ISSUE THREAD

**Issue by rafaelmerini1** [RobinHerbots/jquery.inputmask#2554]
Someone is also having problems with masked fields with numbers on Android?
I can't type numbers right now
In the desktop working normally

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
<script type='text/javascript' src="https://rawgit.com/RobinHerbots/jquery.inputmask/5.x/dist/jquery.inputmask.js"></script>
<script>
$(document).ready(function(){
  $("#pre_celular").inputmask("(99)99999-9999");
});
</script>
<input type="text" class="form-control form-control-lg" name="pre_celular" id="pre_celular">

**Comment by rafaelmerini1** [comment]
https://user-images.githubusercontent.com/91544993/135095017-7592418a-3dc6-419a-b879-e861e794792c.mp4

**Comment by RobinHerbots** [comment]
@rafaelmerini1, 

There is a problem in 5.0.7-beta.18, ... use beta 19 instead.
I know you use rawgit and the latest but apparently it is still pointing to beta 18

**Comment by jvloo** [comment]
> @rafaelmerini1,
> 
> There is a problem in 5.0.7-beta.18, ... use beta 19 instead. I know you use rawgit and the latest but apparently it is still pointing to beta 18

Hi @RobinHerbots , I have updated to 5.0.8-beta.0.. but stil facing the problem as mentioned by @rafaelmerini1 ... May I know if any latest solution available?
